src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,19 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+	public String getPrintableString();
+	
+	/**
+	  * Returns a concatenated version of the printable strings of each object.
+	  */
+	static String getPrintableString(Printable... printables){
+		StringBuilder result = new StringBuilder();
+		for(Printable printable : printables) {
+			if (result.length() != 0) {
```

Comment:
Avoid using magic number. In fact, consider using string.join() here to improve code quality.